homeassistant/components/prometheus/__init__.py

```diff
@@ -70,6 +71,7 @@
             {
                 vol.Optional(CONF_FILTER, default={}): entityfilter.FILTER_SCHEMA,
                 vol.Optional(CONF_PROM_NAMESPACE, default=DEFAULT_NAMESPACE): cv.string,
+                vol.Optional(CONF_API_AUTHENTICATION,default=True): cv.boolean,
```

Comment:
Whitespace missing...
It seems that you skipped the pre-commit hooks :(

Reply:
done